codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    patch:
+      default:
+        target: 80%
```

Comment:
What's the source of this value?

Reply:
uhm seems like the card was edited? pretty sure I took the card suggestion on the coverage and copy-pasted the snippet. I will raise it to 90%